packages/akoya-adapter/src/testInstitutions.ts

```diff
@@ -5,7 +5,7 @@ export const testInstitutions: CachedInstitution[] = [
     name: "Mikomo Bank",
     id: "mikomoBank",
     keywords: ["akoya", "mikomo"],
-    logo: "https://mikomo.ddp.akoya.com/img/default.png",
+    logo: "https://cdn.brandfetch.io/idwKHUTdZK/w/400/h/400/theme/dark/icon.jpeg?c=1bxid64Mup7aczewSAYMX&t=1668071436139",
```

Comment:
I went on a slight tangent and updated some test institution logos...
<img width="376" height="260" alt="image" src="https://github.com/user-attachments/assets/2b03082a-1fff-4a21-a3d1-d0d7a1eae683" />